pecos/xmc/xlinear/model.py

```diff
@@ -144,6 +144,8 @@ def compile_mmap_model(cls, npz_folder, mmap_folder):
         """
         import shutil
 
+        if not path.exists(mmap_folder):
```

Comment:
How about using `os.makedirs(mmap_folder, exist_ok=True)`?
https://docs.python.org/3/library/os.html#os.makedirs

Reply:
I was following the convention of previous changes, in that case should we change https://github.com/amzn/pecos/blob/mainline/pecos/xmc/xlinear/model.py#L98 as well?

Reply:
Updated both lines. Please check revision.